jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -53,8 +53,6 @@ class DataQuality(PluginTemplateMixin, ViewerSelectMixin):
     """
     template_file = __file__, "data_quality.vue"
 
-    irrelevant_msg = Unicode("").tag(sync=True)
```

Comment:
Hmm... this removal seems unrelated?

Reply:
yes, it duplicates the traitlet defined in the mixin.  Would you rather a separate PR for just this?

Reply:
This PR is fine, unless you think it has to be backported. Thanks for the clarification!